.taskcluster.yml

```diff
@@ -82,7 +82,7 @@ tasks:
               /repo/bin/configure_ssh.sh &&
               node /repo/bin/get_secret.js &&
               cd mozilla-central &&
-              ./mach try -p linux64 -u all -t none --artifact
+              ./mach try -b o -p linux64 testing/extensions --artifact
```

Comment:
Hm. I wonder if we could enable a perf test here too, maybe just `-t tp6` to show the capability (that's the pageload test)

Reply:
It's pretty easy to whip up an extension that introduces a big regression in that test :)

Reply:
After a valiant attempt (actually several) to get both perf tests and the custom tests working, it turns out it's not really possible to run specific performance tests (as opposed to _all_ of them, `-t all`, which we don't want per PR) in the same `./mach try` command as custom tests. Issue #19 will address that in a later PR.